docs/src/userguide/language_basics.rst

```diff
@@ -119,40 +187,88 @@ You can read more about them in :ref:`extension-types`.
 Types
 -----
 
-Cython uses the normal C syntax for C types, including pointers.  It provides
-all the standard C types, namely ``char``, ``short``, ``int``, ``long``,
-``long long`` as well as their ``unsigned`` versions, e.g. ``unsigned int``.
-The special ``bint`` type is used for C boolean values (``int`` with 0/non-0
-values for False/True) and ``Py_ssize_t`` for (signed) sizes of Python
-containers.
+.. tabs::
+
+    .. group-tab:: Pure Python
+
+        Cython supports annotaions for C types, including pointers.  It provides
+        all the standard C types, namely ``char``, ``short``, ``int``, ``long``,
+        ``long long`` as well as their ``unsigned`` versions, e.g. ``uint``.
+        The special ``bint`` type is used for C boolean values (``int`` with 0/non-0
+        values for False/True) and ``Py_ssize_t`` for (signed) sizes of Python
+        containers.
+
+        Pointer types are constructed by appending a ``p_`` to the base type
+        they point to, e.g. ``pp_int`` for a pointer to a pointer to a C int.
+        Further pointer types can be constructed with ``pointer()`` function.
```

Comment:
It's false statement, there are "ppp_"  in shadow.py + very important mention is that it's only for interpretation mode, in the compilation mode there are no limit

Reply:
That's not important at all. In real code, pointers will almost never have more than two indirection levels, let alone more than three. Let's be reasonable.

Reply:
@scoder, yeah, you probably would never want to use more then two pointers, but documentation should not lie, at least pp should be changed to ppp, so documentation would convey truth